plugins/time-resources/src/components/PriorityEditor.svelte

```diff
@@ -46,7 +46,7 @@
     }
   })
   $: selected = selectPopupPriorities.find((item) => item.id === value)
-  $: selectedLabel = selected?.label ?? time.string.NoPriority
+  $: selectedLabel = selected?.label === time.string.NoPriority ? time.string.SetPriority : selected?.label
```

Comment:
Shouldn't it be `selected?.label ??  time.string.SetPriority` ?

Reply:
No, I think the tooltip is a hint explaining what button means.
`time.string.NoPriority` is the default value for todo priority.